doc/sample_workflow.md

```diff
@@ -0,0 +1,34 @@
+The lab would like to produce new segmentation datasets using machine learning and inference. The lab has some GPU-accelerated hardware suitable for this use.
+
+# Software prerequisites
+* FFN uses Python 2. It is not presently compatible with Python 3.
+* It is not necessary, but you can install it with "pip install ." so other software can call into it
+* Conda is a reasonable way to get a python2 environment with the components you need
+
+# Setup and Process
+* The area of the global coordinate system that a segmentation dataset covers should be prepared into two hdf5 volumes, one with the raw data and one with the segmentation data. The segmentation data should be in an int64 collection in the volume. The raw data is often in a uint8 collection.
+* Run compute_partitions.py and build_coordinates as described in the readme; these transform the ground truth into a tensorflow record file (used to launch the network for training)
+* Run train.py, giving it the record file under train_coords, as well as the raw image data and the label volume. You should not need to adjust other parameters. This code continually drops new numbered network snapshots; if interrupted it can be restarted. It is best to leave it training for a very long time (even with high-end GPUs, it can be reasonable to leave it running for weeks or more depending on the desired quality). It is not currently capable of using more than one GPU.
+* After train.py is finished, you will have a directory full of model.ckpt-NUMBER.EXTENSION files. Likely you are only interested in the set with the greatest number; this is your trained network.
+* Extract a new, non-overlapping set of data from your image stack (300-edge cubes are reasonable, 600-edge or larger cubes will take awhile but are usable by patient biologists) and save it into an hdf5 file with the same format as the raw data set used for training. This will be your inference target.
+* Look at the pbtxt example in the codebase and make a copy for modification, pointing the image portion to the inference target data you just extracted. Leave the image_mean and image_stddev alone. Set the model_checkpoint_path to model.ckpt-NUMBER (no extension). You likely will want to include a path. Specify a unique segmentation output dir with a name that makes it clear what coordinates you are doing inference against. You may need to tweak some of the inference options (more on this later).
+* When you have that, do run_inference.py, passing it the protobuf-text config you just generated (as in the readme), as well as a bounding box spec (leave the start bit at zero, set the size to be the size of your inference target along each dimension). This will produce results in the directory you specified. It can take awhile
+* When the results are done, you will want to convert them into a format suitable for modification with other software (hdf5, or an image stack). A code snippet to do this is provided below.
+
+# Exporting ffn results
```

Comment:
Perhaps it would be best to make this a standalone script, something like inference_to_hdf5.py?